src/chocolatey.console/Program.cs

```diff
@@ -370,5 +374,24 @@ private static void write_warning_for_incompatible_versions()
 Or by passing the --skip-compatibility-checks option when executing a
 command.");
         }
+
+        private static void check_installed_dotnetfx_version()
+        {
+            if (Platform.get_platform() == PlatformType.Windows)
+            {
+                const string REGKEY = @"SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full\";
+                const int NET48RELEASEBUILD = 528040;

```

Comment:
Where did this number come from? If from documentation on Microsoft site, can you add a link in a comment?

Reply:
Link added.